sdk/src/main/java/com/google/cloud/dataflow/sdk/options/DataflowPipelineWorkerPoolOptions.java

```diff
@@ -145,6 +145,18 @@ public String create(PipelineOptions options) {
   void setNetwork(String value);
 
   /**
+   * GCE <a href="https://cloud.google.com/compute/docs/networking">subnetwork</a> for launching
+   * workers.
+   *
+   * <p>Default is up to the Dataflow service  Expected format is zones/ZONE/subnetworks/SUBNETWORK.
```

Comment:
Done.  For validation, nice to have, but GCE will also send them an error message in short order.